### PR TITLE
Renaissance 2K23 events...

### LINKED ISSUE #4: Implement the Events Component

### PR DESCRIPTION
**Pull request by Cyber-Infernox**
Fixes #4
Implemented the Events component

### CODE REVIEW

File: client/src/components/Events/Events.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
+#container-style {
```
Comment (Ashish-Verma-MNNIT) [inline client/src/components/Events/Events.css:1]: Try to use tailwind wherever possible

File: client/src/components/Events/Events.jsx [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+            {/* <ArrowRightAltIcon className="ml-1" /> */}
+            <button className="bg-gray-300 mt-10 p-2 pl-3 w-40 text-left rounded-sm font-semibold flex justify-left items-center">
+              READ MORE
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke-width="1.5"
+                stroke="currentColor"
+                class="w-6 h-6"
```
Comment (Ashish-Verma-MNNIT) [inline client/src/components/Events/Events.jsx:102]: why two classnames?? 😞  And attributes are camel-cased in jsx.

File: client/src/components/Events/Events.jsx [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+                className="ml-1 w-7 h-6"
+              >
+                <path
+                  stroke-linecap="round"
```
Comment (Ashish-Verma-MNNIT) [inline client/src/components/Events/Events.jsx:106]: Attributes should be camel-cased..

**Review by Ashish-Verma-MNNIT — CHANGES_REQUESTED** [review]
Done a good job. However some changes are needed, look at the comments and make necessary changes....

File: client/src/components/Events/Events.css (hunk 1 above)
Comment (Cyber-Infernox) [reply client/src/components/Events/Events.css]: Done... a903394

File: client/src/components/Events/Events.jsx (hunk 2 above)
Comment (Cyber-Infernox) [reply client/src/components/Events/Events.jsx]: Removed the class attribute... a903394

File: client/src/components/Events/Events.jsx (hunk 3 above)
Comment (Cyber-Infernox) [reply client/src/components/Events/Events.jsx:105]: The attributes were auto generated when I used this SVG.

Comment (sanskaromar) [reply client/src/components/Events/Events.jsx:105]: you can replace them after pasting the SVG code.

File: client/.eslintrc [hunk 4]
```diff
@@ -30,6 +30,7 @@
     "max-depth": ["error", 5], // This rule enforces a maximum depth that blocks can be nested.
     // "multiline-comment-style": ["error", "starred-block"], // This rule enforces a particular style for multiline comments.
     "no-else-return": ["error", { "allowElseIf": false }], // This rule disallows unnecessary blocks following if and else statements.
-    "no-empty": ["error", { "allowEmptyCatch": true }] // This rule disallows empty block statements.
+    "no-empty": ["error", { "allowEmptyCatch": true }], // This rule disallows empty block statements.
+    "prettier/prettier": ["error", { "endOfLine": "auto" }] // To remove errors while reloading the server
```
Comment (sanskaromar) [inline client/.eslintrc:34]: I added the same thing in .prettierrc . Was that not working as expected? If not, then I'll accept these.
As you can notice, comments follow a common syntax "This rule ..." try to rewrite it like that.

**Review by sanskaromar — CHANGES_REQUESTED** [review]

File: client/src/components/Events/Events.jsx (hunk 3 above)
Comment (Cyber-Infernox) [reply client/src/components/Events/Events.jsx]: Done... 891e2f1

File: client/.eslintrc (hunk 4 above)
Comment (Cyber-Infernox) [reply client/.eslintrc]: The terminal was throwing errors. When I added this line in .eslintrc file, the errors were gone.
Changed the comment... 891e2f1

**Review by PriyavKaneria — DISMISSED** [review]
image
For bigger screen sizes if something like this can be done it would improve the overall look. Otherwise nice work

**Comment by NeerajChatterjee** [comment]
> image For bigger screen sizes if something like this can be done it would improve the overall look. Otherwise nice work

Do this for sure as it would look good on Desktop.

**Comment by Cyber-Infernox** [comment]
> image For bigger screen sizes if something like this can be done it would improve the overall look. Otherwise nice work

Done... 79439e9

**Comment by Cyber-Infernox** [comment]
> > image For bigger screen sizes if something like this can be done it would improve the overall look. Otherwise nice work
> 
> Do this for sure as it would look good on Desktop.

Done... 79439e9

**Review by vishwa511 — CHANGES_REQUESTED** [review]
Make them a bit larger. Not just the position , size also matters a lot
you can also use span on first letters on each word and make them more large.
look at Priyav's hand written one and try to make like that
snap33

**Comment by Cyber-Infernox** [comment]
> Make them a bit larger. Not just the position , size also matters a lot you can also use span on first letters on each word and make them more large. look at Priyav's hand written one and try to make like that snap33

Done... fc4d95f

**Review by sanskaromar — APPROVED** [review]

**Pull request merged**